preset/colors/grey.scss

```diff
@@ -1,5 +1,5 @@
 $grey: (
-  'base': #B7C2CB,
+  'base': #D1D8DF,
```

Comment:
So this color already exists and is `grey lighten-1`. These colors are inline with the pattern library. I think if we want to change the border colors for tables it will either be a sass variable override or a style override via `preset/overrides.scss`.

That or you might even just be able to use the `grey lighten-1` class within the app itself.

Reply:
Sorry was rushing a bit, didn't see that lol